### DISCUSSION TITLE
Login on multiple devices simultaneously

### DISCUSSION THREAD

**Discussion by alexjnoble** [nextpyp/.github#10] [Q&A] [answered]
Hello,

Is it possible to allow one user to login on multiple devices simultaneously (browsers, computer, phone, etc)? Currently, when I am logged in through my Firefox browser and then log in on my Chrome browser at the same time, my Firefox nextPYP login then shows the error:

`Not authenticated, access denied`

The same thing happens if I log in from two different computers - the first one gets logged out.

Thanks,
-Alex

**Comment by cuchaz** [comment]
Currently the website is only tracking one session per login account, so when you log in from a new device, the old one will get kicked out. We'll look into what it would take to allow multiple sessions at the same time.

**Comment by abartesaghi** [comment, marked as answer]
Multiple logins have been implemented in [v0.5.3](https://nextpyp.app/files/pyp/0.5.3/docs/index.html)
